scifight/models.py

```diff
@@ -88,6 +88,12 @@ class Problem(models.Model):
     def __str__(self):
         return "#{0}. {1}".format(self.problem_num, self.name)
 
+    def get_short_description(self):
+        if len(self.description) < 90:
+            return self.description
+        else:
+            return self.description[:90] + '...'
```

Comment:
Maybe we should break at least on word boundaries? Sentences like «Read more of that bullshi...» look pretty ugly.